src/test/java/org/beanpod/switchboard/service/StreamLogServiceTest.java

```diff
@@ -47,4 +52,15 @@ final void createStreamLogTest() {
 
     assertEquals(streamLogDto, actualStreamLogDto);
   }
+
+  @Test
+  final void CreateStreamLogCreateStreamLogRequestTest() {
```

Comment:
I'm a bit confused by this test name, maybe `CreateStreamLogFromRequestTest`? But maybe there's something better.

Reply:
This's what happens when you write tests before going to bed :)